ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -299,25 +299,34 @@ impl<'a> TraceBuilder<'a> {
         Ok(())
     }
 
+    fn link_iid_to_last_instr(&mut self, bid: &aot_ir::BBlockId, aot_inst_idx: usize) {
```

Comment:
A docstring may help this function. I was curious as to what it did.

Reply:
I was going to ask the same.

Reply:
I probably should, but this part of trace_builder is a mess, and I hope to somewhat tidy it up soon. Still, fixed in the interim in ea1748d9.

Reply:
Don't worry about this if your next PR will delete this function, but:

> Link the AOT IR to the last instruction pushed into the JIT IR.

The specified AOT IR *BBlockId*, right?

Reply:
Ah never mind.